deploy/02_clusterrole.yaml

```diff
@@ -38,3 +38,47 @@ rules:
       - get
       - list
       - watch
+  # operand resources
+  - apiGroups:
+      - admissionregistration.k8s.io
+    resources:
+      - mutatingwebhookconfigurations
+      - validatingwebhookconfigurations
+    verbs:
+      - create
+      - delete
+      - deletecollection
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - apiextensions.k8s.io
+    resources:
+      - customresourcedefinitions
+    verbs:
+      - create
+      - delete
+      - deletecollection
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - rbac.authorization.k8s.io
+    resources:
+      - clusterroles
+      - clusterrolebindings
+    verbs:
+      - create
+      - bind
```

Comment:
yeah, that shouldn't be.

Reply:
https://github.com/openshift/jobset-operator/pull/1#discussion_r1940950549

Reply:
should be solved by https://github.com/openshift/jobset-operator/pull/1/commits/0420b1f4a229ab21a4db8f6cebdfb4f177865e10